libs/cart/testing/src/drivers/testing/cart-shipping-information/cart-shipping-information.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import {
+  DaffCart,
+  DaffCartShippingInformationServiceInterface,
+  DaffCartShippingRate
+} from '@daffodil/cart';
+
+import { DaffCartFactory, DaffCartShippingRateFactory } from '../../../factories/public_api';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffTestingCartShippingInformationService implements DaffCartShippingInformationServiceInterface<DaffCartShippingRate, DaffCart> {
```

Comment:
You could remove the type definitions for the Interface here and elsewhere, since they all have default values now.

Reply:
changed